elastic/src/main/scala/ElasticFlattened.scala

```diff
@@ -165,13 +167,17 @@ case class ElasticFlattened(M: Monitoring){
   ): ES[Unit] = {
     val E = Executor(Monitoring.defaultPool)
     bufferAndPublish(flaskNameOrHost, flaskCluster)(M, E){ cfg =>
-      val subscription = Monitoring.subscribe(M){ k =>
-        cfg.groups.exists(g => k.startsWith(g))}.map(Option.apply)
 
-      time.awakeEvery(cfg.subscriptionTimeout)(E,
-        Monitoring.schedulingPool).map(_ => Option.empty[Datapoint[Any]]
-      ).wye(subscription)(wye.merge)(E) |>
-        render(environment, flaskNameOrHost, flaskCluster)
+      val data: Process[Task, Option[Datapoint[Any]]] =
+        Monitoring.
+          subscribe(M){ k => cfg.groups.exists(g => k.startsWith(g)) }.
+          map(Option.apply)
+
+      val ticks:                  Process[Task, Duration]               = time.awakeEvery(cfg.subscriptionTimeout)(E, Monitoring.schedulingPool)
+      val ticksAsEmptyDatapoints: Process[Task, Option[Datapoint[Any]]] = ticks.map(_ => Option.empty[Datapoint[Any]])
+      val mergeTicksAndData:      Process[Task, Option[Datapoint[Any]]] = ticksAsEmptyDatapoints.wye(data)(wye.merge)(E)
```

Comment:
I prefer this arrangement (obviously) but I'll allow it is out of the ordinary.  I'll change it.